docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -78,9 +78,18 @@ Below are described the steps you need to make to migrate from v6 to v7.
 - The deprecated props `components` and `componentsProps` have been removed. Use `slots` and `slotProps` instead. See [components section](/x/react-data-grid/components/) for more details.
 - The `slots.preferencesPanel` slot and the `slotProps.preferencesPanel` prop were removed. Use `slots.panel` and `slotProps.panel` instead.
 
-<!-- ### State access
+### State access
 
-- -->
+- Some selectors now require passing `instanceId` as a second argument:
```

Comment:
Would it be helpful to have a list of such selectors in the migration guide, just in case somebody needs to check which ones are affected?